gips/0053-enabling-substreams-based-subgraphs.md

```diff
@@ -0,0 +1,138 @@
+---
+GIP: <0053>
+Title: <Enabling-Substreams-Based-Subgraphs>
+Authors: <Alex Bourget alex@dfuse.io, Adam Fuller adam@edgeandnode.com>
+Created: <2022-07-19>
+Updated: <2023-06-21>
+Stage: <Draft>
+Discussions-To: <https://forum.thegraph.com/t/ggp-0025-enabling-substreams-based-subgraphs/4287, https://forum.thegraph.com/t/substreams-into-subgraphs-a-simple-integration/3542>
+---
+
+# Abstract
+
+Substreams are a new streaming-first blockchain indexing technology developed for The Graph Network, focused on performance and composability.
+
+A substreams package is a collection of interdependent modules, which combine to extract and transform blockchain events in a highly configurable and parallelisable way, producing a streaming output.
+
+This GIP describes the simplest possible integration of Substreams with Subgraphs, to take that final streaming output and make it persistent and queryable.
+
+# Motivation
+
+As Substreams become first-class citizens with The Graph Network, the StreamingFast team believes that Substreams software has been battle tested enough to be deemed fully production ready with The Graph Network. To ensure that Indexers will index substreams-back subgraphs, the GIP proposes to add Oracle support for dataSource.kind =="substreams" to the Feature Matrix shared below. 
+
+This would make an important moment where the performance promises we have made in the last 2 years come to fruition. With a very important indexing-time performance boost, as well as an important injection-time performance boost.
+
+Adding Substream support to Graph Node is the fastest way to bring the performance and composability benefits of substreams to The Graph Network.
+
+# High Level Description
+
+With the launch of Uniswap v3, lots of efforts have been put in stabilizing the engine. In doing so, multiple indeterminism issues were resolved. StreamingFast has done extensive testing for determinism, have validated substreams-graph-load (the high-speed injector) and graph-node loading PoI parity, as well as the gentle hand-off to graph-node with continued parity.
```

Comment:
> With the launch of Uniswap v3

It isn't evident what this means without context

> have validated substreams-graph-load (the high-speed injector)

substreams-graph-load is a separate component which should be considered independently, I don't think it is beneficial to include here

Overall though I think this paragraph would fit more into "validation" than "High level description"

Reply:
Removed Uniswapv3 line. 

Moved remaining line to "Validation"